src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/OffsetOfTests.cs

```diff
@@ -186,8 +186,8 @@ public void OffsetOf_NullType_ThrowsArgumentNullException()
         [Fact]
         public void OffsetOf_NullFieldName_ThrowsArgumentNullException()
         {
-            AssertExtensions.Throws<ArgumentNullException>(null, () => Marshal.OffsetOf(new object().GetType(), null));
-            AssertExtensions.Throws<ArgumentNullException>(null, () => Marshal.OffsetOf<object>(null));
+            AssertExtensions.Throws<ArgumentNullException>("name", () => Marshal.OffsetOf(new object().GetType(), null));
+            AssertExtensions.Throws<ArgumentNullException>("name", () => Marshal.OffsetOf<object>(null));
```

Comment:
If Marshal.OffsetOf were a more entry-level method, I'd advocate taking a change for it to validate the field name directly so it could throw the ANE with the correct parameter name ("fieldName").  But I'm guessing our position right now is "it's an advanced enough API that the caller is expected to realize the parameter sharing between OffsetOf's fieldName and GetField's name."

Writing this down in case someone else is iffy about it and wants to have the discussion. But currently sighing and not recommending further action.